### PR TITLE
Fix release channel

### PR DESCRIPTION
**Pull request by norkans7**
Fix https://github.com/rapidpro/rapidpro/issues/658

### CODE REVIEW

File: temba/channels/migrations/0059_update_nexmo_channels_roles.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ def update_nexmo_channels_roles(apps, schema_editor):
         Channel = apps.get_model('channels', 'Channel')
 
         if settings.IS_PROD:
-            nexmo_channels = Channel.objects.filter(channel_type='NX').exclude(org=None)
+            nexmo_channels = Channel.objects.filter(is_active=True, channel_type='NX').exclude(org=None)
```
Comment (rowanseymour) [inline temba/channels/migrations/0059_update_nexmo_channels_roles.py]: I think only unclaimed android channels can be active without an org, right? so no need for `.exclude(org=None)`

File: temba/channels/models.py [hunk 2]
```diff
@@ -913,7 +913,6 @@ def release(self, trigger_sync=True):
             registration_id = self.gcm_id
 
         # remove all identifying bits from the client
-        self.org = None
```
Comment (rowanseymour) [inline temba/channels/models.py]: CAn we also not clear fields like gcm_id ? Seems good to not throw away information if we don't need to

Comment (norkans7) [reply temba/channels/models.py]: I'll double check on this

Comment (norkans7) [reply temba/channels/models.py]: keeping the gcm_id will make the client keep syncing to the server unnecessary, I think that is the reason of clearing it here.
may be we can dump these values in the config?
@nicpottier @ericnewcomer thoughts?

Comment (rowanseymour) [reply temba/channels/models.py]: seems like if is_active=False then we shouldn't be syncing a channel regardless of GCM?

Comment (norkans7) [reply temba/channels/models.py]: Released channel clients will keep getting the release command if they sync I think that is okay https://github.com/nyaruka/rapidpro/blob/master/temba/channels/views.py#L556

**Pull request merged**